test/suite/hook_tiered.py

```diff
@@ -34,16 +34,24 @@
 #
 # Substitute tiered tables for regular (row-store) tables in Python tests.
 #
-# These hooks can be used to run the existing cursor tests on tiered tables.
-# They identify tests that create row-store tables and create tiered tables
-# instead.  The hook takes an optional argument to specify how many tiers
-# to create. The default is 2.
+# These hooks are intended for basic testing of tiered tables.  There are several
+# pieces of functionality here.
 #
-# To run with 3 tiers per table:
-#     ../test/suite/run.py --hooks tiered=3 cursor
+# 1. We add tiered storage parameters to the config when calling wiredtiger_open()
+#
+# 2. If we create an object that is *not* a table:, we add options to its config
+#    so that it will be stored local-only.  Tiered storage isn't intended (yet?) for
+#    use with LSM or colstore.
```

Comment:
Please expand `colstore` to `column store` (there is/may be another usage in this file too).

Reply:
Done